api/v1alpha2/linodevpc_types.go

```diff
@@ -54,6 +53,9 @@ type VPCSubnetCreateOptions struct {
 	Label string `json:"label,omitempty"`
 	// +optional
 	IPv4 string `json:"ipv4,omitempty"`
+	// ID is subnet id for the subnet
```

Comment:
Should we rename this to SubnetID for clarity similar to how we have VPCID instead of just ID?

Reply:
Yeah, sounds good. Updated